src/nspawn/nspawn.c

```diff
@@ -5687,6 +5718,8 @@ static int run_container(
                                 (void) pty_forward_set_background_color(forward, arg_background);
 
                         set_window_title(forward);
+
+                        pty_forward_set_hotkey_handler(forward, ptyfwd_hotkey, pid);
```

Comment:
Sooner or later `pid` will be converted to `PidRef *` from `pid_t *`, but before moving to that, I'd like to use `PID_TO_PTR(*pid)` here, like the other places
- for consistency with other handlers,
- for safety that `pid` is accidentally freed or changed by other callbacks (when we introduce another callback later and assigning `pid` pointer as is like this).